WordPress-Theme/ruleset.xml

```diff
@@ -99,4 +99,12 @@
 		<message>eval() is a security risk so not allowed.</message>
 	</rule>
 
+	<!--
+		Use WordPress functionality and features first, if available.
+		https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
+	-->
+	<rule ref="WordPress.WP.AlternativeFunctions.file_system_read">
```

Comment:
The error codes have changes with WPCS 0.11.0, so using the group name will no longer work.
You will need to add the error code for each function individually now.

The new codes are (hoping I remember correctly by heart):
* `WordPress.WP.AlternativeFunctions.file_system_read_readfile`
* `WordPress.WP.AlternativeFunctions.file_system_read_fopen`
* `WordPress.WP.AlternativeFunctions.file_system_read_sockopen`
* `WordPress.WP.AlternativeFunctions.file_system_read_pfsockopen`
* `WordPress.WP.AlternativeFunctions.file_system_read_fclose`
* `WordPress.WP.AlternativeFunctions.file_system_read_fread`
* `WordPress.WP.AlternativeFunctions.file_system_read_fwrite`
* `WordPress.WP.AlternativeFunctions.file_system_read_file_put_contents`
* `WordPress.WP.AlternativeFunctions.file_system_read_file_get_contents`